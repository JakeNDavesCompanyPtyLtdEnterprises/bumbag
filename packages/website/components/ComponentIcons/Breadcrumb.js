import React from 'react';

const SvgComponent = props => (
  <svg width="153" height="16" viewBox="0 0 153 16" fill="none" xmlns="http://www.w3.org/2000/svg">
    <path
      d="M5.52 1.24H9.4L4.08 15H0.2L5.52 1.24ZM16.1094 15.24C15.2827 15.24 14.516 15.2267 13.8094 15.2C13.1027 15.1733 12.136 15.1133 10.9094 15.02V1.24H17.7894C18.7627 1.24 19.636 1.38 20.4094 1.66C21.196 1.92667 21.8094 2.31333 22.2494 2.82C22.6894 3.32667 22.9094 3.91333 22.9094 4.58C22.9094 5.40667 22.7294 6.06 22.3694 6.54C22.0094 7.00667 21.4627 7.33333 20.7294 7.52C22.596 7.81333 23.5294 8.91333 23.5294 10.82C23.5294 12.14 22.9227 13.2067 21.7094 14.02C20.5094 14.8333 18.6427 15.24 16.1094 15.24ZM16.9494 6.68C17.4694 6.68 17.896 6.55333 18.2294 6.3C18.576 6.04667 18.7494 5.72 18.7494 5.32C18.7494 4.37333 18.1627 3.9 16.9894 3.9H15.3694V6.68H16.9494ZM16.8694 12.1C17.6027 12.1 18.1627 11.9667 18.5494 11.7C18.9494 11.42 19.1494 11.0333 19.1494 10.54C19.1494 10.0067 18.9694 9.6 18.6094 9.32C18.2494 9.02667 17.736 8.88 17.0694 8.88H15.3494V12.04C15.9494 12.08 16.456 12.1 16.8694 12.1ZM25.0234 5.12H29.2434V7.16C29.5501 6.37333 29.9634 5.79333 30.4834 5.42C31.0168 5.03333 31.6368 4.84 32.3434 4.84L32.9234 8.1C31.6568 8.1 30.7234 8.24 30.1234 8.52C29.5368 8.78667 29.2434 9.21333 29.2434 9.8V15H25.0234V5.12ZM39.1292 15.16C38.0226 15.16 37.0359 14.94 36.1692 14.5C35.3159 14.0467 34.6492 13.4333 34.1692 12.66C33.6892 11.8733 33.4492 10.9867 33.4492 10C33.4492 8.98667 33.6959 8.08667 34.1892 7.3C34.6826 6.51333 35.3492 5.90667 36.1892 5.48C37.0292 5.05333 37.9626 4.84 38.9892 4.84C39.9759 4.84 40.8159 5.06 41.5092 5.5C42.2159 5.92667 42.7426 6.51333 43.0892 7.26C43.4359 7.99333 43.6092 8.82 43.6092 9.74C43.6092 9.98 43.5959 10.1533 43.5692 10.26L37.5692 11.04C37.7292 11.5067 37.9892 11.8467 38.3492 12.06C38.7092 12.26 39.1826 12.36 39.7692 12.36C40.6759 12.36 41.6559 12.1267 42.7092 11.66L43.2092 14.34C41.9692 14.8867 40.6092 15.16 39.1292 15.16ZM40.5092 8.52C40.2959 7.68 39.7892 7.26 38.9892 7.26C38.5092 7.26 38.1292 7.44 37.8492 7.8C37.5692 8.16 37.4226 8.62667 37.4092 9.2L40.5092 8.52ZM49.4825 15.16C48.6158 15.16 47.8025 14.9333 47.0425 14.48C46.2958 14.0133 45.6958 13.3867 45.2425 12.6C44.7892 11.8 44.5625 10.9133 44.5625 9.94C44.5625 8.98 44.7892 8.11333 45.2425 7.34C45.6958 6.55333 46.3025 5.94 47.0625 5.5C47.8225 5.06 48.6292 4.84 49.4825 4.84C50.1758 4.84 50.8158 5.03333 51.4025 5.42C52.0025 5.80667 52.4558 6.38667 52.7625 7.16V4.84H56.9625V15H52.7625V13.14C52.0958 14.4867 51.0025 15.16 49.4825 15.16ZM50.7825 12C51.3158 12 51.7692 11.8133 52.1425 11.44C52.5292 11.0533 52.7358 10.6 52.7625 10.08V9.92C52.7358 9.38667 52.5225 8.93333 52.1225 8.56C51.7358 8.18667 51.2892 8 50.7825 8C50.2225 8 49.7425 8.2 49.3425 8.6C48.9558 8.98667 48.7625 9.45333 48.7625 10C48.7625 10.56 48.9625 11.0333 49.3625 11.42C49.7625 11.8067 50.2358 12 50.7825 12ZM63.3683 15.16C62.4616 15.16 61.6349 14.92 60.8883 14.44C60.1549 13.96 59.5749 13.3267 59.1483 12.54C58.7216 11.74 58.5083 10.8933 58.5083 10C58.5083 9.05333 58.7283 8.18667 59.1683 7.4C59.6216 6.6 60.2216 5.97333 60.9683 5.52C61.7283 5.06667 62.5349 4.84 63.3883 4.84C64.1616 4.84 64.8149 5 65.3483 5.32C65.8949 5.64 66.3349 6.15333 66.6683 6.86V1.82L70.8883 1.2V15H66.6683V13.14C66.0416 14.4867 64.9416 15.16 63.3683 15.16ZM64.6883 12C65.2483 12 65.7149 11.8 66.0883 11.4C66.4749 11 66.6683 10.5333 66.6683 10V9.92C66.6683 9.4 66.4683 8.95333 66.0683 8.58C65.6683 8.19333 65.2083 8 64.6883 8C64.1283 8 63.6549 8.2 63.2683 8.6C62.8949 8.98667 62.7083 9.45333 62.7083 10C62.7083 10.56 62.9016 11.0333 63.2883 11.42C63.6883 11.8067 64.1549 12 64.6883 12ZM81.1841 1.24H85.0641L79.7441 15H75.8641L81.1841 1.24ZM92.6628 15.32C91.3295 15.32 90.1161 15.0067 89.0228 14.38C87.9428 13.74 87.0828 12.8733 86.4428 11.78C85.8161 10.6733 85.5028 9.45333 85.5028 8.12C85.5028 6.76 85.8095 5.53333 86.4228 4.44C87.0361 3.34667 87.8761 2.49333 88.9428 1.88C90.0095 1.25333 91.2095 0.94 92.5428 0.94C93.2628 0.94 93.9695 1.02667 94.6628 1.2C95.3561 1.37333 95.9495 1.62 96.4428 1.94L95.9028 5.04C94.9428 4.68 94.0761 4.5 93.3028 4.5C92.2361 4.5 91.4161 4.81333 90.8428 5.44C90.2695 6.05333 89.9828 6.94 89.9828 8.1C89.9828 9.23333 90.2828 10.12 90.8828 10.76C91.4961 11.3867 92.3428 11.7 93.4228 11.7C93.8361 11.7 94.2228 11.66 94.5828 11.58C94.9428 11.5 95.3828 11.3533 95.9028 11.14L96.4628 14.3C95.3028 14.98 94.0361 15.32 92.6628 15.32ZM97.9531 5.12H102.173V7.16C102.48 6.37333 102.893 5.79333 103.413 5.42C103.946 5.03333 104.566 4.84 105.273 4.84L105.853 8.1C104.586 8.1 103.653 8.24 103.053 8.52C102.466 8.78667 102.173 9.21333 102.173 9.8V15H97.9531V5.12ZM112.199 15.16C111.146 15.16 110.206 14.9733 109.379 14.6C108.566 14.2267 107.932 13.7 107.479 13.02C107.026 12.34 106.799 11.5667 106.799 10.7V4.98H110.919V10.64C110.919 11.08 111.039 11.4267 111.279 11.68C111.532 11.92 111.846 12.04 112.219 12.04C112.579 12.04 112.872 11.92 113.099 11.68C113.339 11.4267 113.459 11.08 113.459 10.64V4.98H117.499V10.7C117.499 11.58 117.272 12.36 116.819 13.04C116.379 13.7067 115.759 14.2267 114.959 14.6C114.159 14.9733 113.239 15.16 112.199 15.16ZM119.438 5.12H123.458V7.16C123.911 6.37333 124.424 5.79333 124.998 5.42C125.584 5.03333 126.278 4.84 127.078 4.84C127.864 4.84 128.544 5.04 129.118 5.44C129.704 5.82667 130.144 6.35333 130.438 7.02C131.291 5.56667 132.531 4.84 134.158 4.84C134.931 4.84 135.604 5.04667 136.178 5.46C136.764 5.87333 137.218 6.47333 137.538 7.26C137.858 8.03333 138.018 8.94667 138.018 10V15H133.798V10C133.798 9.32 133.678 8.80667 133.438 8.46C133.211 8.11333 132.891 7.94 132.478 7.94C132.024 7.94 131.631 8.13333 131.298 8.52C130.978 8.90667 130.818 9.37333 130.818 9.92V15H126.638V10C126.638 9.32 126.518 8.80667 126.278 8.46C126.051 8.11333 125.724 7.94 125.298 7.94C124.844 7.94 124.458 8.13333 124.138 8.52C123.818 8.90667 123.658 9.37333 123.658 9.92V15H119.438V5.12ZM147.505 15.16C145.972 15.16 144.885 14.4867 144.245 13.14V15H140.005V1.82L144.245 1.2V6.86C144.578 6.15333 145.012 5.64 145.545 5.32C146.078 5 146.725 4.84 147.485 4.84C148.378 4.84 149.198 5.06667 149.945 5.52C150.705 5.96 151.298 6.57333 151.725 7.36C152.165 8.14667 152.385 9.02667 152.385 10C152.385 10.96 152.158 11.8333 151.705 12.62C151.265 13.4067 150.665 14.0267 149.905 14.48C149.158 14.9333 148.358 15.16 147.505 15.16ZM146.225 12C146.772 12 147.238 11.8 147.625 11.4C148.012 11 148.205 10.5333 148.205 10C148.205 9.44 148.005 8.96667 147.605 8.58C147.218 8.19333 146.758 8 146.225 8C145.678 8 145.212 8.19333 144.825 8.58C144.438 8.95333 144.245 9.4 144.245 9.92V10C144.245 10.56 144.438 11.0333 144.825 11.42C145.225 11.8067 145.692 12 146.225 12Z"
      fill="#3926A5"
    />
  </svg>
);

export default SvgComponent;
