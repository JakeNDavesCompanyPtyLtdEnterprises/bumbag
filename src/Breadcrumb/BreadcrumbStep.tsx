/* eslint react/prop-types: 0 */
import * as React from 'react';
import * as PropTypes from 'prop-types';

import { BreadcrumbStep as _BreadcrumbStep, BreadcrumbLink as _BreadcrumbLink } from './styled';

export type LocalBreadcrumbStepProps = {
  color?: string;
  href: string;
  isCurrent?: boolean;
  text: string;
};

const BreadcrumbStep: React.FunctionComponent<LocalBreadcrumbStepProps> = ({ color, href, isCurrent, text }) => (
  <_BreadcrumbStep>
    <_BreadcrumbLink aria-current={isCurrent ? 'page' : undefined} color={color} href={href}>
      {text}
    </_BreadcrumbLink>
  </_BreadcrumbStep>
);

BreadcrumbStep.propTypes = {
  color: PropTypes.string,
  isCurrent: PropTypes.bool,
  text: PropTypes.string.isRequired
};

BreadcrumbStep.defaultProps = {
  color: '',
  isCurrent: false
};

// @ts-ignore
const C: React.FunctionComponent<LocalBreadcrumbStepProps> = BreadcrumbStep;
export default C;
