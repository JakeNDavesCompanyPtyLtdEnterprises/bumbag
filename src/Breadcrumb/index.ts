export { default } from './Breadcrumb';
export { default as Breadcrumb } from './Breadcrumb';
