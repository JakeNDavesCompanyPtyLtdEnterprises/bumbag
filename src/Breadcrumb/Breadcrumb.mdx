---
name: Breadcrumb
route: /components/breadcrumb
menu: Components
---

import { Playground } from 'docz';
import PropsTable from '../_docs/components/PropsTable';
import Breadcrumb from './index';


# Breadcrumb

## Import

`import { Breadcrumb } from 'fannypack'`

## Basic Usage

The `<Breadcrumb>` component is used with `<Breadcrumb.Step` to create an accessible Breadcrumb.
  
<Playground>
  <Breadcrumb>
    <Breadcrumb.Step>Home</Breadcrumb.Step>
    <Breadcrumb.Step>Breadcrumb</Breadcrumb.Step>
    <Breadcrumb.Step isCurrent>Basic Usage</Breadcrumb.Step>
  </Breadcrumb>
</Playground>

<Playground>
  <Breadcrumb separationColor="secondary300">
    <Breadcrumb.Step href="/">Home</Breadcrumb.Step>
    <Breadcrumb.Step href="/components/breadcrumb">Breadcrumb</Breadcrumb.Step>
    <Breadcrumb.Step href="components/breadcrumb#basic-usage" isCurrent>Basic Usage</Breadcrumb.Step>
  </Breadcrumb>
</Playground>

## Props

### `<Breadcrumb>`
<PropsTable component={Breadcrumb} />

### `<Breadcrumb.Step>`
<PropsTable component={Breadcrumb.Step} />


## Theming

### Schema

```jsx
{
  base: string | Object,
  Step: {
    base: string | Object
  },
  List: {
    base: string | Object
  },
  Link: {
    base: string | Object
  }
}
```
