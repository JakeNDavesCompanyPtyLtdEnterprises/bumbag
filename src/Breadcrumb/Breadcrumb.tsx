/* eslint react/prop-types: 0 */
import * as React from 'react';
import * as PropTypes from 'prop-types';
import { Breadcrumb as _Breadcrumb, BreadcrumbList as _BreadcrumbList } from './styled';
import BreadcrumbStep, { BreadcrumbStepProps } from './BreadcrumbStep';

export type LocalBreadcrumbProps = {
  children: React.ReactNode;
  separationColor?: string;
};

export type BreadcrumbProps = {
  separationColor?: string;
};

export type BreadcrumbComponents = {
  Step: React.FunctionComponent<BreadcrumbStepProps>;
};

export const Breadcrumb: React.FunctionComponent<LocalBreadcrumbProps> & BreadcrumbComponents = ({
  children,
  separationColor
}) => {
  return (
    <_Breadcrumb a11yTitle="Breadcrumb" separationColor={separationColor}>
      <_BreadcrumbList isOrdered isHorizontal>
        {children}
      </_BreadcrumbList>
    </_Breadcrumb>
  );
};

Breadcrumb.Step = BreadcrumbStep;

Breadcrumb.propTypes = {
  children: PropTypes.node.isRequired,
  separationColor: PropTypes.string
};

Breadcrumb.defaultProps = {
  separationColor: ''
};

// @ts-ignore
const C: React.FunctionComponent<LocalBreadcrumbProps> & BreadcrumbComponents = Breadcrumb;
export default C;
