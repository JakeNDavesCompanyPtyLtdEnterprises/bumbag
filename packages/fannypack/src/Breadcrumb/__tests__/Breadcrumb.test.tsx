import * as React from 'react';
import render from '../../_utils/tests/render';
import Breadcrumb from '../Breadcrumb';

describe('Breadcrumb', () => {
  it('renders correctly', () => {
    const { container } = render(
      <Breadcrumb>
        <Breadcrumb.Step>Home</Breadcrumb.Step>
        <Breadcrumb.Step>Breadcrumb</Breadcrumb.Step>
        <Breadcrumb.Step isCurrent>Basic Usage</Breadcrumb.Step>
      </Breadcrumb>
    );
    expect(container.firstChild).toMatchSnapshot();
  });

  describe('with custom separationColor', () => {
    it('renders correctly', () => {
      const { container } = render(
        <Breadcrumb>
          <Breadcrumb.Step>Home</Breadcrumb.Step>
          <Breadcrumb.Step>Breadcrumb</Breadcrumb.Step>
          <Breadcrumb.Step isCurrent>Basic Usage</Breadcrumb.Step>
        </Breadcrumb>
      );
      expect(container.firstChild).toMatchSnapshot();
    });
  });

  describe('with link', () => {
    it('renders correctly', () => {
      const { container } = render(
        <Breadcrumb>
          <Breadcrumb.Step href="/home">Home</Breadcrumb.Step>
          <Breadcrumb.Step href="/breadcrumb">Breadcrumb</Breadcrumb.Step>
          <Breadcrumb.Step href="/usage" isCurrent>
            Basic Usage
          </Breadcrumb.Step>
        </Breadcrumb>
      );
      expect(container.firstChild).toMatchSnapshot();
    });
  });
});
