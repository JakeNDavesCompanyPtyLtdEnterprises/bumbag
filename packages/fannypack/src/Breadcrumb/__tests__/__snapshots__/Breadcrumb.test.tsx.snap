// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Breadcrumb renders correctly 1`] = `
.c1 {
  margin: unset;
  padding: unset;
  border: unset;
  background: unset;
  font: unset;
  font-family: inherit;
  font-size: 100%;
  box-sizing: border-box;
  background-color: unset;
  color: inherit;
}

.c1:focus:not(:focus-visible) {
  outline: none;
}

.c6 {
  display: inline;
}

.c5 {
  margin-bottom: 0.2rem;
}

.c4 {
  list-style: unset;
  list-style-type: none;
}

.c4:not(:last-child) {
  margin-bottom: 1rem;
}

.c4 .c3 {
  margin-left: 1rem;
}

.c4 {
  list-style-type: decimal;
}

.c4 .c3 {
  list-style-type: lower-alpha;
}

.c4 .c3 .c3 {
  list-style-type: lower-roman;
}

.c4 li {
  display: inline-block;
  margin-right: 1rem;
  margin-bottom: unset;
}

.c0 li + li::before {
  border-right-color: #708da8;
  border-right-style: solid;
  border-right-width: 0.125rem;
  content: '';
  display: inline-block;
  height: 0.8rem;
  margin: 0 1rem;
  -webkit-msrgin-right: 0;
  -moz-msrgin-right: 0;
  -ms-msrgin-right: 0;
  msrgin-right: 0;
  -webkit-transform: rotate(15deg);
  -ms-transform: rotate(15deg);
  transform: rotate(15deg);
}

.c0 [aria-current='page'] {
  font-weight: 600;
  -webkit-text-decoration: none;
  text-decoration: none;
}

.c2 li {
  margin-right: 0;
}

<nav
  aria-label="Breadcrumb"
  class="c0 c1"
>
  <ol
    class="c2 c3 c4 c1"
  >
    <li
      class="c5 c1"
    >
      <span
        class="c6 c1"
      >
        Home
      </span>
    </li>
    <li
      class="c5 c1"
    >
      <span
        class="c6 c1"
      >
        Breadcrumb
      </span>
    </li>
    <li
      class="c5 c1"
    >
      <span
        aria-current="page"
        class="c6 c1"
      >
        Basic Usage
      </span>
    </li>
  </ol>
</nav>
`;

exports[`Breadcrumb with custom separationColor renders correctly 1`] = `
.c1 {
  margin: unset;
  padding: unset;
  border: unset;
  background: unset;
  font: unset;
  font-family: inherit;
  font-size: 100%;
  box-sizing: border-box;
  background-color: unset;
  color: inherit;
}

.c1:focus:not(:focus-visible) {
  outline: none;
}

.c6 {
  display: inline;
}

.c5 {
  margin-bottom: 0.2rem;
}

.c4 {
  list-style: unset;
  list-style-type: none;
}

.c4:not(:last-child) {
  margin-bottom: 1rem;
}

.c4 .c3 {
  margin-left: 1rem;
}

.c4 {
  list-style-type: decimal;
}

.c4 .c3 {
  list-style-type: lower-alpha;
}

.c4 .c3 .c3 {
  list-style-type: lower-roman;
}

.c4 li {
  display: inline-block;
  margin-right: 1rem;
  margin-bottom: unset;
}

.c0 li + li::before {
  border-right-color: #708da8;
  border-right-style: solid;
  border-right-width: 0.125rem;
  content: '';
  display: inline-block;
  height: 0.8rem;
  margin: 0 1rem;
  -webkit-msrgin-right: 0;
  -moz-msrgin-right: 0;
  -ms-msrgin-right: 0;
  msrgin-right: 0;
  -webkit-transform: rotate(15deg);
  -ms-transform: rotate(15deg);
  transform: rotate(15deg);
}

.c0 [aria-current='page'] {
  font-weight: 600;
  -webkit-text-decoration: none;
  text-decoration: none;
}

.c2 li {
  margin-right: 0;
}

<nav
  aria-label="Breadcrumb"
  class="c0 c1"
>
  <ol
    class="c2 c3 c4 c1"
  >
    <li
      class="c5 c1"
    >
      <span
        class="c6 c1"
      >
        Home
      </span>
    </li>
    <li
      class="c5 c1"
    >
      <span
        class="c6 c1"
      >
        Breadcrumb
      </span>
    </li>
    <li
      class="c5 c1"
    >
      <span
        aria-current="page"
        class="c6 c1"
      >
        Basic Usage
      </span>
    </li>
  </ol>
</nav>
`;

exports[`Breadcrumb with link renders correctly 1`] = `
.c1 {
  margin: unset;
  padding: unset;
  border: unset;
  background: unset;
  font: unset;
  font-family: inherit;
  font-size: 100%;
  box-sizing: border-box;
  background-color: unset;
  color: inherit;
}

.c1:focus:not(:focus-visible) {
  outline: none;
}

.c7 {
  margin: unset;
  padding: unset;
  border: unset;
  background: unset;
  font: unset;
  font-family: inherit;
  font-size: 100%;
  box-sizing: border-box;
  background-color: unset;
  color: #444bc9;
}

.c7:focus:not(:focus-visible) {
  outline: none;
}

.c5 {
  margin-bottom: 0.2rem;
}

.c4 {
  list-style: unset;
  list-style-type: none;
}

.c4:not(:last-child) {
  margin-bottom: 1rem;
}

.c4 .c3 {
  margin-left: 1rem;
}

.c4 {
  list-style-type: decimal;
}

.c4 .c3 {
  list-style-type: lower-alpha;
}

.c4 .c3 .c3 {
  list-style-type: lower-roman;
}

.c4 li {
  display: inline-block;
  margin-right: 1rem;
  margin-bottom: unset;
}

.c6 {
  color: #444bc9;
  fill: #444bc9;
  cursor: pointer;
  -webkit-text-decoration: underline;
  text-decoration: underline;
  -webkit-text-decoration-skip: ink edges;
  text-decoration-skip: ink edges;
}

.c6:hover {
  color: #03040b;
  fill: #03040b;
}

.c6 .c8 {
  top: 0.15em;
}

.c0 li + li::before {
  border-right-color: #708da8;
  border-right-style: solid;
  border-right-width: 0.125rem;
  content: '';
  display: inline-block;
  height: 0.8rem;
  margin: 0 1rem;
  -webkit-msrgin-right: 0;
  -moz-msrgin-right: 0;
  -ms-msrgin-right: 0;
  msrgin-right: 0;
  -webkit-transform: rotate(15deg);
  -ms-transform: rotate(15deg);
  transform: rotate(15deg);
}

.c0 [aria-current='page'] {
  font-weight: 600;
  -webkit-text-decoration: none;
  text-decoration: none;
}

.c2 li {
  margin-right: 0;
}

<nav
  aria-label="Breadcrumb"
  class="c0 c1"
>
  <ol
    class="c2 c3 c4 c1"
  >
    <li
      class="c5 c1"
    >
      <a
        class="c6 c7"
        href="/home"
      >
        Home
      </a>
    </li>
    <li
      class="c5 c1"
    >
      <a
        class="c6 c7"
        href="/breadcrumb"
      >
        Breadcrumb
      </a>
    </li>
    <li
      class="c5 c1"
    >
      <a
        aria-current="page"
        class="c6 c7"
        href="/usage"
      >
        Basic Usage
      </a>
    </li>
  </ol>
</nav>
`;
